features/unhandled_view.feature

```diff
@@ -11,7 +11,9 @@ Scenario: Unhandled exceptions are delivered from views
   And the request is a valid for the error reporting API
   And the request contained the api key "a35a2a72bd230ac0aa0f52715bbdc6aa"
   And the payload field "events" is an array with 1 element
-  And the exception "errorClass" equals "ErrorException"
+  And the exception "errorClass" matches one of the following:
+    | ErrorException                              |
+    | Facade\\Ignition\\Exceptions\\ViewException |
```

Comment:
The `errorClass` of unhandled exceptions from Views will now be of the type `Facade\Ignition\Exceptions\ViewException` in Laravel 6 (see: https://laravel-news.com/ignition).  This implementation allows either a type of `ErrorException` or the above mentioned `ViewException`, but could be replaced with a custom and more specific step that verifiers that the `errorClass` is correct for the particular Laravel version.

Reply:
I think this is sufficient as it asserts that an error is handled - I don't see the value in testing that Laravel is throwing the right one for the version.